lib/sqlalchemy/orm/query.py

```diff
@@ -3414,8 +3428,8 @@ class BulkDelete(BulkUD):
     """BulkUD which handles DELETEs."""
 
 
-class RowReturningQuery(Query[Row[_TP]]):
+class RowReturningQuery(Query[Row[Unpack[_Ts]]]):
     if TYPE_CHECKING:
 
-        def tuples(self) -> Query[_TP]:  # type: ignore
+        def tuples(self) -> Query[tuple[Unpack[_Ts]]]:  # type: ignore
```

Comment:
**Michael Bayer** (zzzeek) wrote:

still on python 3.8 for minimum version so this should be Tuple

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Done